bluebrain/repo-patches/packages/steps/package.py

```diff
@@ -68,7 +68,7 @@ class Steps(CMakePackage):
     depends_on("mpi", when="+mpi")
     depends_on("petsc~debug+int64+mpi", when="+petsc+mpi")
     depends_on("petsc~debug+int64~mpi", when="+petsc~mpi")
-    depends_on("pkg-config", type="build")
+    depends_on("pkgconf", type="build")
```

Comment:
```suggestion
    depends_on("pkgconfig", type="build")
```
As the providers section lists that as the virtual provider:
```
      pkgconfig: [pkgconf, pkg-config]
```
Then either `pkgconf` can satisfy that by default, or `pkg-config` if the user really insists.